404.html

```diff
@@ -0,0 +1,16 @@
+---
+# example 404.md
+
+layout: default
+permalink: /404.html
+---
+
+# 404
+
+Page not found! :(
+
+<script>
+    if( window.location.pathname.indexOf('/docs/') >= 0){
+        window.location = '/docs/v3';
```

Comment:
Could this not be `window.location = '/docs/v3' + window.location.pathname.substr(5)`, so that we take the user directly to the path they need to get to rather than the root.

Reply:
Good Call! Will get this added.